src/SprykerSdk/Spryk/SprykConfig.php

```diff
@@ -138,6 +154,11 @@ public function getDefaultDevelopmentMode(): string
         return static::NAME_DEVELOPMENT_LAYER_PROJECT;
     }
 
+    public function getDefaultLevel(): int
```

Comment:
Too ambiguous. You have default value in console command and config loader and they are different from each other.

Reply:
clarified the name of the method